.ci/atime/tests.R

```diff
@@ -75,96 +75,20 @@ test.list <- atime::atime_test_list(
-    Regression = "c152ced0e5799acee1589910c69c1a2c6586b95d", # Parent of the first commit (https://github.com/Rdatatable/data.table/commit/15f0598b9828d3af2eb8ddc9b38e0356f42afe4f) in the PR (https://github.com/Rdatatable/data.table/pull/4558/commits) that fixes the regression
-    Fixed = "f750448a2efcd258b3aba57136ee6a95ce56b302"), # Second commit of the PR (https://github.com/Rdatatable/data.table/pull/4558/commits) that fixes the regression
-
-  # Issue with sorting again when already sorted: https://github.com/Rdatatable/data.table/issues/4498
-  # Fixed in: https://github.com/Rdatatable/data.table/pull/4501
-  "DT[,.SD] improved in #4501" = atime::atime_test(
-    N = 10^seq(1, 10, by=0.5),
-    setup = {
-      set.seed(1)
-      L = as.data.table(as.character(rnorm(N, 1, 0.5)))
-      setkey(L, V1)
-    },
-    ## New DT can safely retain key.
-    expr = data.table:::`[.data.table`(L, , .SD),
-    Fast = "353dc7a6b66563b61e44b2fa0d7b73a0f97ca461", # Close-to-last merge commit in the PR (https://github.com/Rdatatable/data.table/pull/4501/commits) that fixes the issue 
-    Slow = "3ca83738d70d5597d9e168077f3768e32569c790", # Circa 2024 master parent of close-to-last merge commit (https://github.com/Rdatatable/data.table/commit/353dc7a6b66563b61e44b2fa0d7b73a0f97ca461) in the PR (https://github.com/Rdatatable/data.table/pull/4501/commits) that fixes the issue 
-    Slower = "cacdc92df71b777369a217b6c902c687cf35a70d"), # Circa 2020 parent of the first commit (https://github.com/Rdatatable/data.table/commit/74636333d7da965a11dad04c322c752a409db098) in the PR (https://github.com/Rdatatable/data.table/pull/4501/commits) that fixes the issue 
-
-  # Issue reported in: https://github.com/Rdatatable/data.table/issues/6286
-  # Fixed in: https://github.com/Rdatatable/data.table/pull/6296
-  "DT[by, verbose = TRUE] improved in #6296" = atime::atime_test(
-    N = 10^seq(1, 9),
-    setup = {
-      dt = data.table(a = 1:N)
-      dt_mod <- copy(dt)
+      df <- data.frame(x = runif(N), y = rnorm(N), z = sample(letters, N, replace = TRUE))
+      dt <- as.data.table(df)
     },
-    expr = data.table:::`[.data.table`(dt_mod, , 1, by = a, verbose = TRUE),
-    Slow = "a01f00f7438daf4612280d6886e6929fa8c8f76e", # Parent of the first commit (https://github.com/Rdatatable/data.table/commit/fc0c1e76408c34a8482f16f7421d262c7f1bde32) in the PR (https://github.com/Rdatatable/data.table/pull/6296/commits) that fixes the issue
-    Fast = "f248bbe6d1204dfc8def62328788eaadcc8e17a1"), # Merge commit of the PR (https://github.com/Rdatatable/data.table/pull/6296) that fixes the issue
+    expr = data.table:::`[.data.table`(transform(dt, xRounded = round(x), ySquared = y^2, zLength = nchar(z), conditional = ifelse(y > 0, x * y, x / y))),
+    Slow = "64e2041ad48838679ac4c901a04557f5a7ba2df3",
+    Fast = "cde7333938a590a3cddbda1b02103650e2f55d15",
+    Next = "18a7209b242fa2a784fe76be845106baacd8349f",
+    Next2 = "2d1a0575f87cc50e90f64825c30d7a6cb6b05dd7",
```

Comment:
On https://github.com/Rdatatable/data.table/commit/2d1a0575f87cc50e90f64825c30d7a6cb6b05dd7 we can see this is where the commit was merged into master, so this should be the Fast commit.
We can also see on that page that bf499090c0e6fd5cb492bf8b1603d93c1ee21dfb is its parent, which should be used as the Slow commit. @Anirban166 Can you please close this and submit a PR to Rdatatable/data.table?

Reply:
Yes I did previously use those SHAs as Fast and Slow respectively for the same reason (just needed to use `transform.data.table`), I added the other SHAs because they were also commits in master with mentions from that PR (back then I was thinking in case one of them actually made a difference)

Sure! I will send a PR to Rdatatable/data.table now and close the issue in atime!

Reply:
Yes I did previously use those SHAs as Fast and Slow respectively for the same reason (check my comment above saying 'Merged to master in: https://github.com/Rdatatable/data.table/commit/2d1a0575f87cc50e90f64825c30d7a6cb6b05dd7') but I added the other SHAs because they were also commits in master with mentions from that PR (back then, I was thinking in case one of them actually makes a difference)

Sure! I will send a PR to Rdatatable/data.table now and close the issue in atime!